google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java

```diff
@@ -162,6 +166,9 @@ private Builder(com.google.api.services.bigquery.model.JobConfiguration configur
         this.clustering = Clustering.fromPb(loadConfigurationPb.getClustering());
       }
       this.useAvroLogicalTypes = loadConfigurationPb.getUseAvroLogicalTypes();
+      if (configurationPb.getLabels() != null) {
+        this.labels = configurationPb.getLabels();
+      }
     }
```

Comment:
I think you don't need to check configurationPb.getLabels() != null.
this.labels set to null by default, so null is a valid value.